Jenkinsfile

```diff
@@ -16,13 +16,17 @@ pipeline {
     productionHost = credentials('PRODUCTIONHOST')
   }
   parameters {
+    choice(choices: ['validate', 'create'], description: 'Behaviour at connection time (initialize/validate schema)',
```

Comment:
Can you add other choices according to https://docs.spring.io/spring-boot/docs/current/reference/html/howto-database-initialization.html. It would be good to maintain the order according to each choice's likelihood of being used.

Reply:
I guess validate should be the only option , but to ease deployment for demo/testing purposes i added create.

Reply:
Although `validate` may be most frequently used, there are other choices like `update` (e.g. for adding a new column/constraint) could also be useful. What I meant was to make all avaialbe options for the `spring.jpa.hibernate.ddl-auto` a choice here. We may need one of them at some point in the future.

Reply:
I also wonder if all the options should be available in all the environments. For instance, using an option that would drop and recreate the production database could be **extremely** dangerous. It would be preferable to synchronize a staging database into production, and using only `validate` in production.

Reply:
If no better option out there, we can apply this choice only to staging DB and make production environments always use `validate`. I think `update` option is probably as useful as, if not more useful than, `create`for the staging. Sometime, there is only small "update" to the schema and you want to keep the data already loaded.

Reply:
Yes, that looks good to me.

Reply:
i can do that but until we release or set up production ready metadata api ,
1. We will be testing the production only for demo purposes so we need to use the create for production to ease the deployment . 
2.Other option is make the staging public for testing purposes and implement the above ddl options in that way the prod ddl should be manually configured.
as far i understand option 2 looks good . i can make that way.

Reply:
Yes, the one we should use for demo/beta purposes is staging (wwwdev).